_drafts/simple-strategies-to-manage-techdebt-part-1.md

```diff
@@ -272,7 +272,7 @@ As a recap, these are five examples of technical debt appears:
  - Because new tools / technology / patterns emerge that solves problems in a different and more efficient way
  
 We never consider our bugs technical debt items (unless they become a [feature](https://www.wired.com/story/its-not-a-bug-its-a-feature/)),
-but we might consider third party bugs, problems or missing features and other external things that we might need invest
+but we might consider third party bugs, problems or missing features and other external things that we might needto invest
```

Comment:
Hah Got ya! you also made a typo =)

No need to fix, I'll do myself on merge =)
```suggestion
but we might consider third party bugs, problems or missing features and other external things that we might need to invest
```